packages/javascript-sdk/src/http-client/helpers.ts

```diff
@@ -31,13 +31,13 @@ export function addAuthzInfoToHeaders(
   const headers = new Headers(init.headers);
 
   if (advices.AuthenticateToServiceConditionAdvice) {
-    headers.set('x-tree', advices.AuthenticateToServiceConditionAdvice[0]);
+    headers.set('X-Tree', advices.AuthenticateToServiceConditionAdvice[0]);
   } else if (advices.TransactionConditionAdvice) {
-    headers.set('x-txid', advices.TransactionConditionAdvice[0]);
+    headers.set('X-Txid', advices.TransactionConditionAdvice[0]);
```

Comment:
I'm getting nothing useful from backstage searching for "X-Txid" or even "Txid", so I just wanted to ask a similar questions as "Accept-Api-Version" below - is our strategy here just to capitalise the first letter of each word here?

Reply:
Anything that starts with `x-` is a customer header, so there's no right or wrong way to do this. I can make a change to be a bit more consistent at least internally.